test/EventStore.Client.PersistentSubscriptions.Tests/Bugs/Issue_1125.cs

```diff
@@ -57,12 +57,7 @@ public static IEnumerable<object[]> TestCases() => Enumerable.Range(0, 50)
 						// This is a retry
 						await subscription.Ack(@event);
 					}
-				}, (s, dr, e) => {
-					if (e != null)
-						completed.TrySetException(e);
-					else
-						completed.TrySetException(new Exception($"{dr}"));
-				}, userCredentials)) {
+				}, userCredentials: userCredentials)) {
```

Comment:
How come removing the dropped callbacks? (here and other places)

Reply:
Search and replace gone wrong